src/backend/util2.c

```diff
@@ -54,6 +54,8 @@ void util_assert(const char *file, int line)
 {
     fflush(stdout);
     printf("Internal error: %s %d\n",file,line);
+    printf("This is a compiler bug, please report it via "
+        "https://issues.dlang.org/enter_bug.cgi\n");
```

Comment:
1. should be major, not critical.
2. I wonder if having this link would encourage spammers to post on bugzilla by raising the visibility of it